src/components/EditableHeading/EditableHeading.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Why not fix it and be done with it?

Reply:
@sahariko this component needs too be refactored IMO

Reply:
Alright, it's not a blocker or anything.